actions/workflow.jupyter.user.delete.yaml

```diff
@@ -0,0 +1,27 @@
+name: workflow.jupyter.user.delete
```

Comment:
Could we call this something like:
`workflow.jupyter.user.cleanup` to not confuse it with our delete action